CTP/sql/src/com/navercorp/cubridqa/cqt/console/util/PropertiesUtil.java

```diff
@@ -102,6 +102,13 @@ public static void initConfig(String configFile, Test test) throws Exception {
 			test.setHoldcas(val);
 		}
 
+		List sol = root.selectNodes(TestUtil.ROOT_NODE + TestUtil.SERVER_OUTPUT);
+		if (!hc.isEmpty()) {
```

Comment:
It must be sol, instead of hc.

Reply:
I omitted to push the change. thank you.

Reply:
```suggestion
		if (!sol.isEmpty()) {
```